test/conftest.py

```diff
@@ -69,9 +97,26 @@ def db_uri(request, config):
 
 
 @pytest.fixture(scope='session')
-def connection(db_uri):
+def is_postgresql(db_uri):
+    if POSTGRESQL_DB in db_uri:
```

Comment:
It might actually be more confusing to have this defined as a constant than simply used inline. There is only one use, and it's not likely to change ;)

Reply:
That's true. I'll change it.